conf/bk_server.conf

```diff
@@ -706,6 +706,7 @@ ledgerDirectories=/tmp/bk-data
 # dbStorage_rocksDB_numLevels=-1
 # dbStorage_rocksDB_numFilesInLevel0=4
 # dbStorage_rocksDB_maxSizeInLevel1MB=256
```

Comment:
Should this be left empty instead, to use the default value?

Reply:
This is a commented out example value, the default behavior has not changed. Please suggest a different suggested, commented out value if desired.

Reply:
I forgot that it was commented :). In general we keep all the keys commented with their default value, which in this case is empty.

Reply:
Agree, it's better to keep empty to indicate the default value.